dabs.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+docker run -it --rm -v `pwd`:/abs/workdir/ app_build_suite:latest $@
```

Comment:
Backticks are legacy syntax

```suggestion
docker run -it --rm -v "$(pwd)":/abs/workdir/ app_build_suite:latest $@
```